sources/osgDB/Input.js

```diff
@@ -39,6 +41,15 @@ define( [
         //     imageLoadingUsePromise: undefined,
         //     imageOnload: undefined,
         // };
+
+        var channelCtor = function () {};
```

Comment:
You could move this code in osgAnimation.js wrapper at the begining of the file and do something like:

```
osgDB.registry().registerObject('osgAnimation.Vec3LerpChannel', channelCtor );
osgDB.registry().registerObject('osgAnimation.FloatLerpChannel', channelCtor );
```